dashboard/src/shared/url.ts

```diff
@@ -9,6 +9,10 @@ export const app = {
   },
 };
 
+function withNS(namespace: string) {
+  return namespace === "_all" ? "" : `namespaces/${namespace}`;
```

Comment:
If you made this `namespaces/${namespace}/` (ie. so that the non-all case has a trailing slash), then you could avoid the double-slash below when namespace is `_all`?